src/test/java/io/confluent/idesidecar/restapi/resources/ConfluentQueryResourceTestBase.java

```diff
@@ -134,7 +138,32 @@ void setupCCloudApiMocks(String bearerToken) {
         bearerToken,
         "ccloud-resources-mock-responses/get-schema-registry-empty.json"
     );
-  }
+
+    // Register mock for listing Flink compute pools
+    ccloudTestUtil.expectSuccessfulCCloudGet(
+        computePoolListUri.formatted(mainTestEnvId),
+        bearerToken,
+        "ccloud-resources-mock-responses/list-flink-compute-pools-empty.json"
+    );
+    // Register mock for listing Flink compute pools within ccloud request
+    ccloudTestUtil.expectSuccessfulCCloudGet(
+        computePoolListUri.formatted(emptyEnvId),
+        bearerToken,
+        "ccloud-resources-mock-responses/list-flink-compute-pools-empty.json"
+    );
+    // Register mock for listing Flink compute pools
+    ccloudTestUtil.expectSuccessfulCCloudGet(
+        computePoolListUri.formatted(mainTestEnvId),
+        bearerToken,
+        "ccloud-resources-mock-responses/list-flink-compute-pools.json"
+    );
+    // Register mock for listing Flink compute pools within ccloud request
+    ccloudTestUtil.expectSuccessfulCCloudGet(
+        computePoolListUri.formatted(emptyEnvId),
+        bearerToken,
+        "ccloud-resources-mock-responses/list-flink-compute-pools.json"
+    );
```

Comment:
Why do we register each compute pool URL (one with the main test env id and another one with the empty env id) twice?